packages/ember-views/lib/views/view.js

```diff
@@ -764,14 +764,16 @@ var View = CoreView.extend(
     @property layout
     @type Function
   */
-  layout: computed(function(key) {
+  layout: computed('layoutName', function(key, value) {
```

Comment:
Need to detect if using feature flag for new-computed and use alternate syntax, otherwise a deprecation will be thrown and the tests will fail.